src/eatery/campus_eatery.py

```diff
@@ -0,0 +1,89 @@
+import requests
+
+from src.constants import (
+    STATIC_MENUS_URL,
+    TRILLIUM_ID,
```

Comment:
Trailing comma here but not on lines 9 and 19

Reply:
Any way we could use a linter to deal with these kinds of stylistic requirements? @meganle

Reply:
Good catch, we use flake8. Let's see if we can add rule to the config to prohibit trailing commas so that Hound catches this stuff @cswenberg @orkosinha